ExampleApp/ExampleApp/Screens/CentralServices/CentralServiceCell.swift

```diff
@@ -0,0 +1,63 @@
+import UIKit
+
+class CentralServiceCell: UITableViewCell {
+
+    static let reuseId: String = "service-cell"
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setupLayout()
+    }
+
+    required init?(coder: NSCoder) { nil }
+
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        uuidLabel.text = nil
+        isPrimaryLabel.text = nil
+        characterisicsCountLabel.text = nil
+    }
+
+    // MARK: - Subviews
+
+    let uuidLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.boldSystemFont(ofSize: 14.0)
+        return label
+    }()
+
+    let isPrimaryLabel: UILabel = {
+        let label = UILabel()
+        label.text = "isPrimary: --"
+        return label
+    }()
+
+    let characterisicsCountLabel: UILabel = {
+        let label = UILabel()
+        label.text = "charac. count: --"
+        return label
+    }()
+
+    let stackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.spacing = 10.0
+        return stackView
+    }()
+
+    // MARK: - Private
+
+    private func setupLayout() {
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        contentView.addSubview(stackView)
+        [uuidLabel, isPrimaryLabel, characterisicsCountLabel].forEach(stackView.addArrangedSubview)
```

Comment:
why not just: `stackView.addArrangedSubviews([uuidLabel, isPrimaryLabel, characterisicsCountLabel])`

Reply:
I don't have this extension here :) (see `UIStackView+addArrangedSubviews.swift` in 'one of our other projects') :p

Reply:
;)